src/cswinrt/code_writers.h

```diff
@@ -10230,19 +10230,16 @@ return true;
             return;
         }
 
-        if (is_flags_enum(type))
-        {
-            w.write("[FlagsAttribute]\n");
-        }
-
         auto enum_underlying_type = is_flags_enum(type) ? "uint" : "int";
 
-        w.write(R"(%%%% enum % : %
+        w.write(
+R"(
+%[WindowsRuntimeMetadata("Windows.Foundation.FoundationContract")]
+[global::System.ComponentModel.EditorBrowsable(global::System.ComponentModel.EditorBrowsableState.Never)]
+% enum % : %
 {
-)", 
-        bind<write_winrt_attribute>(type),
-        bind<write_winrt_exposed_type_attribute>(type, false),
-        bind<write_type_custom_attributes>(type, true),
+)",             
+        is_flags_enum(type) ? "[FlagsAttribute]\n" : "",
         (settings.internal || settings.embedded) ? (settings.public_enums ? "public" : "internal") : "public",
```

Comment:
we might want to keep write_type_custom_attributes.  Not sure if it wrote anything relevant for enums or not, but its basically any attribute someone decided to put on the enum in the winmd.